bin.js

```diff
@@ -179,6 +180,21 @@ cli
   )
   .action(Coupon.issue)
 
+  cli
+  .command('bridge generate-tokens <did>')
+  .option('-c, --can', 'One or more abilities to delegate.')
+  .option(
+    '-e, --expiration',
+    'Unix timestamp when the delegation is no longer valid. Zero indicates no expiration.',
```

Comment:
```suggestion
    'Unix timestamp (in seconds) when the delegation is no longer valid. Zero indicates no expiration.',
```

Reply:
whoops - when I scanned this PR I somehow missed these comments - will address and submit a followup PR right now!